### ISSUE TITLE
Family Link accounts reportedly can't sign in

### ISSUE THREAD

**Issue by cdsmith** [google/codeworld#787]
I have a report that users of Family Link accounts (this is Google's system for limited accounts for children) cannot log in to CodeWorld.  I should look into this.

**Comment by cdsmith** [comment]
I'v confirmed that if I create a Family Link account, it cannot log in to CodeWorld.  I get the message "Looks like your Google Account can't go here" after logging in.

I'm not sure what is possible to do about this.  I also tried Khan Academy, and it offered a flow for a parent to approve the child account to log in, so that looked promising and indicates there's some way for parents to approve third-party apps.  But after I approved, it still answers "Can't visit this page.  It's not available for your account."

I will do some more research into what integrations are possible here.

**Comment by rybesh** [comment]
Thanks for looking into this. I spent some time chatting with a Google One / Google for Families support person, but unfortunately they were completely useless—they couldn't understand the difference between visiting a website, and using Google as an OAuth provider to sign into a website.

**Comment by rybesh** [comment]
tl;dr: I guess CodeWorld currently does not “meet the right web filters for children,” whatever those might be. 

Support drone #3382678 reports:

> Your child can sign in to most Android apps by Google using their Google Account. They won't be able to sign in with their account on any third-party apps and games (e.g., Pokemon Go), or any apps by Google that are still not available to children like Wear.  

> On the other hand, Google does not own and control third-party developers apps and websites. If their content settings do not meet the right web filters for children, the websites and apps will not be available.

**Comment by cdsmith** [comment]
I've given up on finding external documentation, and it's not looking promising.  It turns out, though, that I work on Google's education products for a day job, and maybe I can get more complete information from one of the product managers I work with.  I'll keep you posted.

**Comment by FlavienBusseuil** [comment]
I'm having the same problem here and looking for documentation/informations. Did you found anything?

**Comment by briff** [comment]
It seems as of today all third party app access became unavailable for managed child accounts, including code.world, though it did work the day before.
"Whoops! You're using an account that can't go here." error is displayed.

**Comment by cdsmith** [comment]
Oh, that's not great.  Unfortunately, I no longer work at Google, and even when I did I had no luck trying to track down the policy around these accounts.

**Comment by edwindevaan** [comment]
Hi, have you retried third party app access? It seems Google has re-enabled it. See [https://support.google.com/families/answer/9204736](url)
